Description: add a useful whatis entry to the generated man page
Origin: vendor
Forwarded: no
Author: Ryan Niebur <ryan@debian.org>
Reviewed-by: gregor herrmann <gregoa@debian.org>
Last-Update: 2011-06-06


--- a/xs/GtkMozEmbed.xs
+++ b/xs/GtkMozEmbed.xs
@@ -567,7 +567,7 @@
 
 # --------------------------------------------------------------------------- #
 
-=for object Gtk2::MozEmbed::main
+=for object Gtk2::MozEmbed::main - Perl interface to the Mozilla embedding widget
 
 =head1 SIGNALS
 
